### PR TITLE
Testing/test setup

### PR DESCRIPTION
**Pull request by GraceGardner**
What (if any) features are you implementing?
- Cypress set up
- Add basic fixtures
- Creates tests for Manage, ScoreCard and Search components/functionality 
- Add classes to elements for testing purposes 
- Fixed syntax error in Manage.js 

What (if anything) did you refactor?
N/A

Were there any issues that arose?
N/A

Is there anything that you need from your teammate?
Remember to npm install to run cypress 

Any other comments, questions, or concerns?
Not at this time

### CODE REVIEW

File: cypress/integration/Manage_spec.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+describe('Manage league component', () => {
+
+  beforeEach(() => {
+    cy.intercept('Get', 'https://discify-api.herokuapp.com/api/v1/players',
+    { fixture: 'all-players.json'})
+
+    cy.intercept('Post','https://discify-api.herokuapp.com/api/v1/players', {
+      "id": 1,
+      "name": "Hannah",
+      "average_score": 0,
```
Comment (philalewis) [inline cypress/integration/Manage_spec.js:10]: If this is adding players, I think we just need the name and id.

File: cypress/integration/score_card_spec.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+describe('Scorecard user flow', () => {
+
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/')
+      .get('.search-btn').click().click()
```
Comment (philalewis) [inline cypress/integration/score_card_spec.js:5]: I don't know why we have to click twice here. It was doing the same thing yesterday.

**Review by philalewis — APPROVED** [review]
Just a couple of things in the tests. When we ad players, we only need a name and id in the stubbed data. Also, I would like to figure out why we have to clikc the search button twice in order to search.

**Review by oacortez — APPROVED** [review]
Thank you for creating this branch Grace! I see we created and started basic implementation of cypress testing. I see we also implemented some data into the cypress such as all course, single course, and players. Not too sure if we need that, since I think we can do the same when we fetch a get API inside a cypress test file. Maybe something we can look into. Other than that, everything looks great. I will merge this branch. Thank you!

**Pull request merged**